contrib/systemd/cilium

```diff
@@ -23,6 +23,9 @@ PATH=/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
 # Check more cli options using cilium-agent -h
 
 CILIUM_OPTS=--kvstore consul --kvstore-opt consul.address=127.0.0.1:8500
+CILIUM_OPERATOR_OPTS=--kvstore consul --kvstore-opt consul.address=127.0.0.1:8500 --k8s-kubeconfig-path=/home/vagrant/.kube/config
```

Comment:
Is there a reason this is in the `cilium` file instead of a separate file for cilium operator?

Reply:
No particular reason.